kubernetes-csi-external-attacher.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: kubernetes-csi-external-attacher
+  version: 4.7.0
+  epoch: 0
+  description: Sidecar container that watches Kubernetes VolumeAttachment objects and triggers ControllerPublish/Unpublish against a CSI endpoint
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-csi/external-attacher
+      tag: v${{package.version}}
+      expected-commit: ac93ce17a8a602f9310085a86b88a72230231257
+
+  - uses: go/build
+    with:
+      packages: ./cmd/csi-attacher
+      ldflags: "-s -w -X main.version=v${{package.version}} -extldflags '-static'"
+      output: csi-attacher
+
+  - uses: strip
```

Comment:
We can omit using `-s` because we need symbols and debug info. `-` is part of `go/build` pipeline already. `- uses: strip` can be omitted too.